admin/metrics/client.go

```diff
@@ -98,3 +98,101 @@ func (c *Client) AutoscalerSlotsRecommendations(ctx context.Context, limit, offs
 
 	return recommendations, nil
 }
+
+type Usage struct {
+	OrgID      string    `json:"org_id"`
+	ProjectID  string    `json:"project_id"`
+	StartTime  time.Time `json:"start_time"`
+	EndTime    time.Time `json:"end_time"`
+	MetricName string    `json:"metric_name"`
+	Amount     float64   `json:"amount"`
```

Comment:
Should we maybe call this `value` or `max_value` instead?

Reply:
changed to value as agg func can change as per the metric